CGPs/cgp-0040.md

```diff
@@ -0,0 +1,52 @@
+---
+cgp: 40
+title: Reduce Exchange Spreads for cUSD and cEUR
+date-created: 2021-09-30
+author: Roman Croessmann (@rcroessmann), Martin Volpe (@martinvol), Nadiem Sissouno (sissnad), Zviad Metreveli (@zviadm), Markus Franke (@MarkusBerlin)
+status: DRAFT
+date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+governance-proposal-id: [if submitted]
+---
+## Overview
+
+This governance proposal suggests a reduction of the fee charged for cEUR and cUSD exchanges via Mento, the Celo stability protocol.
+
+Since the activation of cUSD and cEUR via governance proposals [CGP007](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0007.md) and [CGP24](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0024.md), the Celo Reserve has charged a fee of 0.5%, a value controlled via the `spread` parameter in the Exchange contract, on all trades with the Celo Reserve. This fee is supposed to compensate the Celo Reserve for taking on price risk as a liquidity provider.
+
+### Status
+- DRAFT = Feedback collection
+
+A 0.5% spread effectively leads to an arbitrage free range on the cUSD/USD and cEUR/EUR rate of at least 0.5% in both directions. Additional fees an arbitrageur encounters, like fees at centralized exchanges, gas costs etc., further increase the range in which cUSD/USD and cEUR/EUR can fluctuate before arbitrage incentives are present that create pressure towards an exchange rate of 1:1.
+
+With the sharp increase in cUSD and cEUR demand seen over the last few weeks, the price of cUSD and cEUR has, as expected, been trading mainly at the upper bound of this arbitrage free zone, staying below 1.01 most of the time. To tighten the peg further, we suggest reducing the spread to 0.25% which can be expected to reduce the arbitrage free range by about 0.25% in each direction. The downside of a spread reduction is a lower fee for the reserve for any given trade as well as a larger loss for the reserve in case of an imprecise oracle rate. We however believe that the expected tighter peg of cUSD and cEUR and expected larger total trading amounts more than outweigh these potential downsides.
```

Comment:
Have we tested these scenarios? ie imprecise oracle rate and 0.25% spread

Reply:
We looked at different spreads and noisy oracle rates in the simulation in the past. IMO the bottom line is that the Mento bucket sizes (influenced by the `reserveFraction` parameter) is the major driver determining reserve risk due to imprecise oracle rates whereas the spread is far less important.

Reply:
Please continue discussions here: https://forum.celo.org/t/discussions-for-cgp-40-proposal-to-reduce-exchange-spreads/1619